dspace/config/crosswalks/symplectic_xwalks_dspace_default.xsl

```diff
@@ -54,63 +49,46 @@
         <crosswalks:mapping dspace="dcterms.references" elements="references" />
-        <!-- <crosswalks:mapping dspace="pubs.author-url" elements="author-url" /> -->
         <crosswalks:mapping dspace="dc.relation.uri" elements="author-url,publisher-url,url" />
-        <!-- <crosswalks:mapping dspace="pubs.are-files-confidential" elements="are-files-confidential" /> -->
         <crosswalks:mapping dspace="pubs.awarded-date" elements="awarded-date" />
         <crosswalks:mapping dspace="pubs.commissioning-body" elements="commissioning-body" />
-        <!-- <crosswalks:mapping dspace="pubs.confidential" elements="confidential" /> -->
-        <!-- <crosswalks:mapping dspace="pubs.edition" elements="edition" /> -->
         <crosswalks:mapping dspace="dc.description.version" format-elements="~[$edition] (Edition)~" />
-        <!-- <crosswalks:mapping dspace="pubs.embargo-release-date" elements="embargo-release-date" />-->
-        <!-- <crosswalks:mapping dspace="pubs.is-embargoed" elements="is-embargoed" /> -->
         <crosswalks:mapping dspace="pubs.filed-date" elements="filed-date" />
         <crosswalks:mapping dspace="pubs.finish-date" elements="finish-date" />
-        <!--<crosswalks:mapping dspace="pubs.pii" elements="pii" />-->
         <crosswalks:mapping dspace="pubs.patent-number" elements="patent-number" />
         <crosswalks:mapping dspace="pubs.patent-status" elements="patent-status" />
         <crosswalks:mapping dspace="pubs.place-of-publication" elements="place-of-publication" />
-   <!-- <crosswalks:mapping dspace="pubs.publisher-url" elements="publisher-url" /> -->
-   <!-- <crosswalks:mapping dspace="pubs.publication-status" elements="publication-status" /> -->
         <crosswalks:mapping dspace="pubs.repository-status" elements="repository-status" />
         <crosswalks:mapping dspace="pubs.start-date" elements="start-date" />
     </crosswalks:mappings>
     
-   <!-- <dim:field mdschema="pubs" element="organisational-group" qualifier="">
-        replaces '/' with blank space
-        <xsl:value-of select="translate(., '/', ' ')" />
-    </dim:field> -->
     
     <!--
         Object level field mappings
         ===========================
         Map values from the publication fields into repository fields
     -->
     <crosswalks:object-mappings for="dspace">
-        <!--<crosswalks:mapping dspace="pubs.requested-embargo-period" elements="p-requested-embargo-period" />
-            <crosswalks:mapping dspace="pubs.notes" elements="p-requested-embargo-period" /> -->
             <crosswalks:mapping dspace="dc.description.notes" elements="fulltext-comment" />
+            <crosswalks:mapping dspace="dc.description.notes" elements="c-invited" />
             These mappings are for the OA location selection
-        <!--<crosswalks:mapping dspace="dc.identifier.uri" elements="p-oa-location" /> -->
+            <crosswalks:mapping dspace="dc.identifier.url" elements="p-oa-location" />
```

Comment:
@mello99 What is the Elements "p-oa-location" field in human-readable terms? I don't see it in the "Other" item type.